packages/vite/src/node/preview.ts

```diff
@@ -135,6 +141,9 @@ export async function preview(
     config,
     middlewares: app,
     httpServer,
+    async close() {
+      await closeHttpServer()
+    },
```

Comment:
Personally I feel we don't need the intermediate variable, but really a nit
```suggestion
    close: createServerCloseFn(httpServer),
```

Reply:
Done.